### DISCUSSION TITLE
Private chat as a movable entity

### DISCUSSION THREAD

**Discussion by Antwannnnn** [runelite/runelite#15022] [Ideas]
Is the private chat a movable part of the game interface? If so, it would be great to be able to move it independently, along with the rest of the UI

![unknown](https://user-images.githubusercontent.com/106710995/171515490-d89fe8f5-8622-40f7-96cc-405e5b68d2cd.png)

**Comment by pkadep** [comment]
please make this, i bought a 27 inch monitor and cant play with chat on left so i moved big chunk of chat to the right, and my split chat is all the way on the left

**Comment by pkadep** [comment]
![image](https://user-images.githubusercontent.com/71012546/195995764-aecffd91-66fd-40e5-bd6f-a23c7729392f.png)

**Comment by Adam-** [comment]
The root layer for the pm box is the size of the entire canvas, so it isn't something which could be moved unless we rewrote how the interface works to not be so large.
